### PR TITLE
[7/N] Fix Wextra-semi warning

### PR DESCRIPTION
**Pull request by cyyever**
Fixes #ISSUE_NUMBER

cc @H-Huang @awgu @kwen2501 @wanchaol @fegin @fduwjj @wz337 @wconstab @d4l3k @c-p-i-o @EikanWang @jgong5 @wenzhe-nrv @sanchitintel @mingfeima @XiaobingSuper @ashokei @jingxu10

### CODE REVIEW

**Review by ezyang — APPROVED** [review]

File: torch/csrc/jit/runtime/static/ops.cpp [hunk 1]
```diff
@@ -676,9 +673,9 @@ REGISTER_OPERATOR_FUNCTOR(
              &clamp_min,
              &clamp_max,
              &nan,
-             &output_size});
+             &output_size})
       };
-    });
```
Comment (kit1980) [inline torch/csrc/jit/runtime/static/ops.cpp:681]: @cyyever this is actually failing internally now 

```
torch/csrc/jit/runtime/static/ops.cpp:677:7: error: expected ';' after expression
    2024-11-11 07:33:09,694 [PID 4349] ERROR      677 |       };
    2024-11-11 07:33:09,694 [PID 4349] ERROR          |       ^
    2024-11-11 07:33:09,694 [PID 4349] ERROR    1 error generated.
```

**Pull request closed without merging**